sdk/spring/azure-spring-boot-starter-storage/README.md

```diff
@@ -20,11 +20,11 @@ The project provides a Spring Boot Starter `azure-spring-boot-starter-storage` t
 ```
 
 ## Key concepts
-The Azure Storage platform is Microsoft's cloud storage solution for modern data storage scenarios. Core storage services offer a massively scalable object store for data objects, disk storage for Azure virtual machines (VMs), a file system service for the cloud, a messaging store for reliable messaging, and a NoSQL store.
+The [Azure Storage platform][azure_storage] is Microsoft's cloud storage solution for modern data storage scenarios. Core storage services offer a massively scalable object store for data objects, disk storage for Azure virtual machines (VMs), a file system service for the cloud, a messaging store for reliable messaging, and a NoSQL store.
```

Comment:
nit: how about using `[Azure Storage platform]` instead of `[Azure Storage platform][azure_storage]`.

Reply:
I see